case_studies/psf_variation/conf/config.yaml

```diff
@@ -0,0 +1,321 @@
+---
+defaults:
+    - _self_
+    - override /hydra/job_logging: stdout
+
+# completely disable hydra logging
+# https://github.com/facebookresearch/hydra/issues/910
+hydra:
+    output_subdir: null
+    run:
+        dir: .
+
+paths:
+    sdss: /data/scratch/sdss
+    decals: /data/scratch/decals
+    des: /data/scratch/des
+    dc2: /data/scratch/dc2local
+    cached_data: null  # TODO: override in other configs
+    output: ${oc.env:HOME}/bliss_output
+
+# this prior is sdss-like; the flux parameters were fit using SDSS catalogs
+prior:
+    _target_: bliss.simulator.prior.CatalogPrior
+    survey_bands: ["u", "g", "r", "i", "z"]  # SDSS available band filters
+    reference_band: 2  # SDSS r-band
+    star_color_model_path: ${simulator.survey.dir_path}/color_models/star_gmm_nmgy.pkl
+    gal_color_model_path: ${simulator.survey.dir_path}/color_models/gal_gmm_nmgy.pkl
+    n_tiles_h: 20
+    n_tiles_w: 20
+    tile_slen: 4
+    batch_size: 64
+    max_sources: 1
+    mean_sources: 0.01  # 0.0025 is more realistic for SDSS but training takes more iterations
+    min_sources: 0
+    prob_galaxy: 0.5144
+    star_flux_exponent: 0.4689157382430609
+    # star_flux_truncation: 613313.768995269
+    star_flux_truncation: 1000
+    star_flux_loc: -0.5534648001193676
+    star_flux_scale: 1.1846035501201129
+    galaxy_flux_exponent: 1.5609458661807678
+    # galaxy_flux_truncation: 28790.449063519092
+    galaxy_flux_truncation: 1000
+    galaxy_flux_loc: -3.29383532288203
+    galaxy_flux_scale: 3.924799999613338
+    galaxy_a_concentration: 0.39330758068481686
+    galaxy_a_loc: 0.8371888967872619
+    galaxy_a_scale: 4.432725319432478
+    galaxy_a_bd_ratio: 2.0
+
+surveys:
+    sdss:
+        _target_: bliss.surveys.sdss.SloanDigitalSkySurvey
+        dir_path: ${paths.sdss}
+        fields:
+            - run: 94
+              camcol: 1
+              fields: ${range:12,482,20}
```

Comment:
Would you change `base_config.yaml` to use multiple fields by default? (That will also cause pytest to use multiple fields by default, so maybe two is enough.) Going forward, because of the success of your project, we basically always want to train with multiple PSFs.

Reply:
I added `3900/6/269` as we have it saved in `tests/data/sdss`.

Reply:
Great!